generator/lib/src/class_element_ex.dart

```diff
@@ -149,7 +152,8 @@ extension ClassElementEx on ClassElement {
 
       final objectType = ObjectType.values[modelInfo.value.getField('type')!.getField('index')!.toIntValue()!];
 
-      final mappedFields = fields.realmInfo.toList();
+      // Computed fields go last. This is important for the schema generation.
```

Comment:
```suggestion
      // Realm Core requires computed properties at the end so we sort them at generation time versus doing it at runtime every time. 
```

Reply:
@nielsenko added this one for the generator. Imo, we should be more clear why we are doing the sort there